walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -121,6 +123,70 @@ public void everyAttendeeIsConsidered() {
     Assert.assertEquals(expected, actual);
   }
 
+  @Test
+  public void everyAttendeeIsConsideredExceptOptionalAttendee() {
+    // Have each person have different events. Have an optional attendee who has an entire day of events
+    // We should see three options because each person has split the restricted times.
+    // We should not consider the optional attendee
+    //
+    // Events  :       |--A--|     |--B--|
+    // Optional: |------------C----------------|
+    // Day     : |-----------------------------|
+    // Options : |--1--|     |--2--|     |--3--|
+
+    Collection<Event> events = Arrays.asList(
+        new Event("Event 1", TimeRange.fromStartDuration(TIME_0800AM, DURATION_30_MINUTES),
+            Arrays.asList(PERSON_A)),
+        new Event("Event 2", TimeRange.fromStartDuration(TIME_0900AM, DURATION_30_MINUTES),
+            Arrays.asList(PERSON_B)),
+        new Event("Event 3", TimeRange.WHOLE_DAY,
+            Arrays.asList(PERSON_C)));
+
+    MeetingRequest request =
+        new MeetingRequest(Arrays.asList(PERSON_A, PERSON_B), DURATION_30_MINUTES);
+    request.addOptionalAttendee(PERSON_C);
+
+    Collection<TimeRange> actual = query.query(events, request);
+    Collection<TimeRange> expected =
```

Comment:
nit: I believe set up for testing is usually preferred to be: (1) set up, (2) act, (3) assert
so, (1) set up everything you need, (2) trigger the actual act, then (3) assert that your result is as expected
I think you mostly follow that here, but I would move this `expected` to the same section where you define the meetingrequest, and THEN call query.query etc